package.json

```diff
@@ -24,7 +24,7 @@
     "prettier": "^2.5.1",
     "prettier-plugin-prisma": "3.8.0",
     "pretty-quick": "3.1.3",
-    "turbo": "latest",
+    "turbo": "1.0.24",
```

Comment:
Related issue: https://github.com/vercel/turborepo/issues/549

Reply:
That issue was (supposedly) fixed in v1.1.0 ([ref](https://github.com/vercel/turborepo/issues/549#issuecomment-1022284349))

Reply:
Cool! thanks for letting me know!